core/vm.c

```diff
@@ -36,8 +36,8 @@
 #include "include/ept.h"
 #include "include/paging.h"
 
-static uint8_t vm_mid_bits = 0;
-#define VM_MID_BIT 8
+static uint64_t vm_mid_bits = 0;
+#define VM_MID_BIT 64
```

Comment:
Could you please put this define into `core/include/config.h`?

I need it for the NetBSD backend. Also HAX_MAX_VCPUS is a little bit more verbose to me rather than cryptic VM_MID_BIT.

Reply:
Hi. I changed HAX_MAX_VCPUS to 128.

Reply:
Hmm, now `VM_MID_BIT` is defined twice in two different files. The ideal solution is to define it once in one common header (`config.h`, or another one) and include that header in `vm.c`. Also, I agree with @krytarowski that we should take the chance to rename it to `HAX_MAX_VMS` (I believe `HAX_MAX_VCPUS` is a typo).

@insidiaGithub You could go ahead with the proper refactoring. Or, if you want this PR to focus on the original issue, that's also fine. Please keep only the first commit, and I'll approve it. @krytarowski If #137 can't handle 64 VMs, you can define a separate constant in a NetBSD header and limit it to 8.